src/main/java/com/woowacourse/matzip/application/RestaurantService.java

```diff
@@ -65,4 +67,17 @@ public RestaurantResponse findById(final Long restaurantId) {
                         .orElseThrow(RestaurantNotFoundException::new)
         );
     }
+
+    public RestaurantTitlesResponse findTitlesByCampusIdAndNameContainingIgnoreCaseIdDescSort(final Long campusId,
```

Comment:
서비스 메서드의 이름은 검색한다라는 의미로 줄여도 좋을것 같은데 어떠세요?

```suggestion
    public RestaurantTitlesResponse searchRestaurantsByTitle(final Long campusId,
```